### PR TITLE
Add new typos

### PR DESCRIPTION
**Pull request by shirayu**
I added new typos by using [GitHub Typo Corpus](https://github.com/mhagiwara/github-typo-corpus).

The code to find typos is [shirayu/find_new_typos](https://github.com/shirayu/find_new_typos).
(You can set another parameters to find new typos.)

### CODE REVIEW

File: crates/typos-dict/assets/words.csv [hunk 1]
```diff
@@ -146,6 +147,7 @@ absailing,abseiling
 absance,absence
 abscence,absence
 abscound,abscond
+absed,based
```
Comment (epage) [inline crates/typos-dict/assets/words.csv:150]: Can you update this to `absed,abased,based`

File: crates/typos-dict/assets/words.csv [hunk 2]
```diff
@@ -16600,6 +16754,7 @@ diverstiy,diversity
 divertion,diversion
 divertions,diversions
 divet,divot
+diviation,deviation
```
Comment (epage) [inline crates/typos-dict/assets/words.csv:16757]: Could you update this to `diviation,divination,deviation`

**Review by epage — COMMENTED** [review]
> The code to find typos is [shirayu/find_new_typos](https://github.com/shirayu/find_new_typos).
(You can set another parameters to find new typos.)

This is great!  I'd be willing to have this integrated so we can update from this more often

File: crates/typos-dict/assets/words.csv (hunk 1 above)
Comment (shirayu) [reply crates/typos-dict/assets/words.csv:150]: Updated at [cb37366](https://github.com/crate-ci/typos/pull/530/commits/cb3736663e61ab7d97839481696bae0c77434865)

File: crates/typos-dict/assets/words.csv (hunk 2 above)
Comment (shirayu) [reply crates/typos-dict/assets/words.csv:16757]: Updated at [cb37366](https://github.com/crate-ci/typos/pull/530/commits/cb3736663e61ab7d97839481696bae0c77434865)

**Comment by epage** [comment]
Alright, 1.11 is now out!

**Pull request merged**